### PR TITLE
Added Elm filetype support

### PR DESCRIPTION
**Pull request by bigardone**
These changes add support for [Elm](http://elm-lang.org/) filetype.

### CODE REVIEW

File: tests/list_file_types.t [hunk 1]
```diff
@@ -65,6 +65,9 @@ Language types are output:
   
     --elixir
         .ex  .eex  .exs
+
```
Comment (jschpp) [inline tests/list_file_types.t]: You need to add whitespace here.

**Comment by bigardone** [comment]
@jschpp done :)

**Comment by jschpp** [comment]
Now you only need to remove the whitespace in front of this comma:
```diff
-    { "elm" , { "elm" } },
+    { "elm", { "elm" } },
```
:)

**Comment by bigardone** [comment]
@jschpp sorry about that :(
done!

**Comment by ahstro** [comment]
Bumping this, since such an easy merge has to have been forgotten? :)

**Pull request merged**